boards/arm/sam_v71_xult/sam_v71_xult-common.dtsi

```diff
@@ -46,7 +46,7 @@
 	pwmleds {
 		compatible = "pwm-leds";
 		pwm_led0: pwm_led_0 {
-			pwms = <&pwm0 0 PWM_MSEC(20)>;
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
```

Comment:
Look at [commit-guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines) about to automatically link a commit with an issue., since this should close #51544.